lib/jekyll/commands/new.rb

```diff
@@ -21,6 +22,12 @@ def init_with_program(prog)
         def process(args, options = {})
           raise ArgumentError, "You must specify a path." if args.empty?
 
+          if args.size == 1
+            new_blog_title = args[0]
```

Comment:
I know it's not the "UNIX" thing to do, but can you `.upcase` the title here? Lowercase directory names just don't look right.

Yes, it's a nitpick, but it's _my_ nitpick 😏